tests/documents.py

```diff
@@ -50,6 +51,7 @@ class Meta:
             'launched',
             'type',
         ]
+        doc_type = 'car_document'
```

Comment:
@HansAdema, is this change necessary? Seems redundant with the use of `@car_index.doc_type`? I've tried this PR in my own code without adding `doc_type` to `Meta` and it seems to work fine.